Samsung.voice recorder 1.0.1.38 2091f478 8dda 4b05 9533 d2479975e59a

upintheairsheep opened the pull request:
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113299&drop=dogfoodAlpha

upintheairsheep committed:
New package: Samsung.VoiceRecorder version 1.0.1.38

upintheairsheep committed:
Update manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

upintheairsheep committed:
Update manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

upintheairsheep committed:
Update manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

upintheairsheep committed:
Update manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

stephengillie commented:
Automatic Validation ended with:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/4c7764d2-593e-4e16-a29f-1f058488f66d)

upintheairsheep committed:
Use official name and description from SWDesc.txt

;LangID Lang / Export to
;0412 KOR / KOR
;0409 ENG/UK, HKG
;040C FRN/FRN
;0407 GER/GER
;0411 JPN / JPN
;0404 CHT / CHT
;0804 CHS/CHS
;0C0A SPA / SPA
;0419 RUS / RUS
;0816 POR/POR
;0410ITA/ITA

[Default]
Section=0409

[0412] ; KOR/KOR
ItemName=Samsung Voice Recorder
ItemDesc= WMA/WAV 형식의녹음또는재생을위한프로그램입니다

[0409] ; ENG/UK, HKG
ItemName=Samsung Voice Recorder
ItemDesc=A program to record and play files in the WMA/WAV format.

[040C]; FRN/FRN
ItemName=Samsung Voice Recorder
ItemDesc=Programme qui enregistre et joue des fichiers au format WMA/WAV.

; GER/GER
ItemName=Samsung Voice Recorder
ItemDesc=Ein Programm zum Aufzeichnen und Wiedergeben von Dateien im Format WMA/WAV.

[OC0A] ; SPA / SPA
ItemName=Samsung Voice Recorder
ItemDesc=Un programa para grabar y reproducir archivos en el formato WMA/WAV.

[0816] ; POR/POR
ItemName=Samsung Voice Recorder
ItemDesc=Um programa para gravar e reproduzir ficheiros em formato WMA/WAV.

[0419] ; RUS/RUS
ItemName=Samsung Voice Recorder
ItemDesc=Программа для записи и воспроизведения файлов в формате WMA/WAV.

[0404] ; CHT/CHT
ItemName=Samsung Voice Recorder
ItemDesc=A program for recording and playing WMA/WAV format files.

[0804] ; CHS/CHS
ItemName=Samsung Voice Recorder
ItemDesc=A program for recording and playing WMA/WAV format files.

[0410] ; ITA/ITA
ItemName=Samsung Voice Recorder
ItemDesc=Un programma per registrare e riprodurre file in format to WMA/WAV.

upintheairsheep committed:
Update Samsung.VoiceRecorder.installer.yaml

stephengillie commented:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/f65f2ecd-36ba-4b7b-a92a-9b65f1b1c296)

Clicking OK starts the installer. I think the manifest isn't getting the package installed correctly.

upintheairsheep committed:
Update Samsung.VoiceRecorder.installer.yaml

stephengillie commented:
Automatic Validation ended with:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/193f8b6e-74dc-459c-b2fc-243e8d3aaf25)

upintheairsheep commented:
> Automatic Validation ended with: ![image](https://user-images.githubusercontent.com/114952565/260164569-193f8b6e-74dc-459c-b2fc-243e8d3aaf25.png)

Interactive Only?
